data/calculated_parser/functions.py

```
@@ -69,11 +70,468 @@ def sspeed(depth, latitude, temperature, salinity):
+
+    # Find total_idx
+    total_idx = count_numerical_vals(speed) - 1  #speed.size - np.count_nonzero(np.isnan(speed)) - 1
+    
+    # Create Layer Subset
+    lower_subset = speed[sca_idx + 1: total_idx + 1]
+
+    if lower_subset.size == 0:
+        return np.nan
+
+    # Find min and max value of subset
+    min_value = np.nanmin(lower_subset)
```

Ruff (0.12.2)

Local variable `min_value` is assigned to but never used

Remove assignment to unused variable `min_value`

(F841)